src/abstractions/request-builder.service.ts

```diff
@@ -0,0 +1,13 @@
+import { OrganizationImportRequest } from "@/jslib/common/src/models/request/organizationImportRequest";
+
+import { GroupEntry } from "@/src/models/groupEntry";
+import { UserEntry } from "@/src/models/userEntry";
+
+export abstract class RequestBuilderAbstratction {
```

Comment:
`Abstraction` suffix is not required:

```suggestion
export abstract class RequestBuilder {
```

(then your names should be good!)